downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/BiologicalProcessAnnotationBuilder.java

```diff
@@ -118,25 +117,25 @@ private static void processProteins(Set<GKInstance> proteinInstances, GKInstance
     /**
      * Before creating GOA line for BP annotations, the reaction in question needs to be checked for the existance of a 'goBiologicalProcess' attribute. If there is none
      * than the instance's 'hasEvent' referrals are checked for any.
-     * @param referenceEntityInst -- ReferenceEntity instance from protein instance.
-     * @param reactionInst
-     * @param taxonIdentifier -- CrossReference ID of protein's species.
-     * @throws Exception
+     * @param referenceEntityInst -- GKInstance, ReferenceEntity instance from protein instance.
+     * @param reactionInst -- GKInstance, parent reaction instance.
+     * @param taxonIdentifier -- String, CrossReference ID of protein's species.
+     * @throws Exception -- MySQLAdaptor exception.
      */
     private static void getGOBiologicalProcessLine(GKInstance referenceEntityInst, GKInstance reactionInst, String taxonIdentifier) throws Exception {
         for (Map<String, String> biologicalProcessAccession : getGOBiologicalProcessAccessions(reactionInst, 0)) {
-            String goaLine = GOAGeneratorUtilities.generateGOALine(referenceEntityInst, BIOLOGICAL_PROCESS_LETTER, biologicalProcessAccession.get("accession"), biologicalProcessAccession.get("event"), "TAS", taxonIdentifier);
+            String goaLine = GOAGeneratorUtilities.generateGOALine(referenceEntityInst, GOAGeneratorConstants.BIOLOGICAL_PROCESS_LETTER, biologicalProcessAccession.get(GOAGeneratorConstants.ACCESSION_STRING), biologicalProcessAccession.get(GOAGeneratorConstants.EVENT_STRING),  GOAGeneratorConstants.TRACEABLE_AUTHOR_STATEMENT_CODE, taxonIdentifier);
             GOAGeneratorUtilities.assignDateForGOALine(reactionInst, goaLine);
         }
     }
 
     /**
      * This method checks for a populated 'goBiologicalProcess' attribute in the incoming instance. If there are none and the max recursion has been reached,
      * it's 'hasEvent' referral is checked for it. Once finding it, it returns the 'accession' and 'identifier' for each one, which will be used to generate a GOA line.
-     * @param eventInst -- Can be the original reaction instance, or, if it had no Biological Process accessions, its Event referrals.
-     * @param recursion -- Indicates number of times the method has been recursively called.
-     * @return
-     * @throws Exception
+     * @param eventInst -- GKInstance, Can be the original reaction instance, or, if it had no Biological Process accessions, its Event referrals.
+     * @param recursion -- Integer, Indicates number of times the method has been recursively called.
```

Comment:
For clarity, it might be better to write the type as "int" instead of "Integer" since it is a primitive.